bentoml/pytorch.py

```diff
@@ -67,20 +71,15 @@ def load(
         booster = bentoml.pytorch.load(
             'lit_classifier:20201012_DE43A2', device_id="cuda:0")
     """  # noqa
-    model = model_store.get(tag)
-    weight_file = model.path_of(f"{SAVE_NAMESPACE}{PT_EXT}")
+    model_info = model_store.get(tag)
+    weight_file = model_info.path_of(f"{SAVE_NAMESPACE}{PT_EXT}")
     # TorchScript Models are saved as zip files
     if zipfile.is_zipfile(weight_file):
```

Comment:
perhaps in a follow-up PR, could we store the model type somewhere in the model metadata YAML instead of relying on file format? that way is a bit more future proof - say if there's a new model format for PyTorch, and if it also uses zip file format. If it's well structured in the metadata file, we don't need to worry about backward compatibility.

Reply:
cc @larme